docs/source/models/intervention_models/mncnh_pregnancy/iv_iron_antenatal/iv_iron_mncnh.rst

```diff
@@ -89,28 +88,46 @@ Hemoglobin exposure
 
 .. todo::
 
-  Update IV iron effect size to be consistent with new data from Chris T.
-  Also, assume no individual level heterogeneity despite having some data on this. (We chose not to model this in order to simplify the data prep for this model)
+  - Decide how we want to model uncertainty in the effect size of IV iron on hemoglobin concentrations.
```

Comment:
Google AI gave me a pretty good answer to the following question: "how do i compute a 95% confidence interval around a mean difference in paired data?". Let's use those instructions (or similar) and see if the studies report sufficient information. Let me know if you want to discuss in more detail!